pom.xml

```diff
@@ -164,15 +165,16 @@
         <phoenix.version>4.7.0.2.5.0.0-1245</phoenix.version>
         <redis.lettuce.version>3.4.2.Final</redis.lettuce.version>
         <scala.version>2.10.5</scala.version>
-        <!-- Ideally we need to use official version of SR: wait for new release -->
-        <schema-registry.version>0.6.0.3.4.0.0-145</schema-registry.version>
+        <!-- This version should be changed to actual HDF-3.4.0.0-146 version when it is published -->
+        <schema-registry.version>0.6.1-SNAPSHOT-HDF-3.4.0.0-146-maybe</schema-registry.version>
```

Comment:
Should be changed to 3.4.0.0-146 (or higher) or new official SR version.

Reply:
Agree to keep the HDF version until the new SR version is pushed.